e2e/data-factory/run-test.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Simple script to run the database connection test
+
+echo "Running Data Factory Database Connection Test..."
+echo "=============================================="
+
+# Change to the e2e directory
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)